scripts/capture.py

```diff
@@ -0,0 +1,31 @@
+import cv2
+import datetime
+import os
+
+OUTPUT_DIR = '/home/pi/data'
+DIR = '%Y-%m-%d'
+FILE = '%H:%M:%S.png'
+
+cap = cv2.VideoCapture(0)
+
+#  warmup camera
+for i in range(15):
+    _, _ = cap.read()
+
+ret, frame = cap.read()
```

Comment:
@Adman you should always check the return value. It will save you from lot of headaches.